### DISCUSSION TITLE
Lesson 3: I can't deploy a contract with payable function

### DISCUSSION THREAD

**Discussion by af66** [smartcontractkit/full-blockchain-solidity-course-py#1865] [Q&A]
Hello everyone, 

I'm at the beginning of the lesson 3, and I'm already having trouble, I spent 1 hour trying some things so I decided to ask for your help.

With that simple code I have a problem:

```
// SPDX-License-Identifier: MIT
pragma solidity >=0.6.6 <0.9.0;

contract FundMe{

    function Fund() public payable{


    }
}
```

If i try to deploy the contract, it's fine. But as soon as I type something else than 0 in the "Value" field  :

https://www.noelshack.com/2023-15-2-1681228558-image-2023-04-11-175557860.png

I always have this error: 

```
creation of FundMe pending...
creation of FundMe errored: VM error: revert.

revert
	The transaction has been reverted to the initial state.
Note: The called function should be payable if you send value and the value you send should be less than your current balance.
Debug the transaction to get more information.
[vm]from: 0x5B3...eddC4to: FundMe.(constructor)value: 1 weidata: 0x608...30033logs: 0hash: 0xdb7...26d98
```

I don't understand this problem as I'm using a test account with a lot of Ether on it. 

I'm doing the exact same thing than Patrick in the course but it doesn't work

Can someone help me ? Thanks for your help.

**Comment by cromewar** [comment]
Hello @af66 
This might be due to how REMIX manages the functions on the new implementation, I suggest you to complete the functionality of the function before continuing, and also be sure you are using the correct accounts on the REMIX id.
